test/index.js

```diff
@@ -1124,6 +1124,52 @@ describe('expect()', () => {
                 Hoek.assert(!exception, exception);
                 done();
             });
+
+            const tests = [
```

Comment:
This isn't what I meant. One `it()` block. No `tests` array. No loop. You can just use curly braces to define a scope and do a simple test inside of it so you can reuse the same variable names. See https://github.com/nodejs/node/blob/4a408321d9c4a6964c9d89a0dd09067f36b4dbfc/test/parallel/test-vm-debug-context.js#L34-L72 as an example.